packages/components/organisms/f-checkout/src/tenants/en-AU.js

```diff
@@ -1,4 +1,223 @@
-export default {
+const messages = {
     locale: 'en-AU',
-    text: 'I am a VueCheckout Component (AU)'
+
+    labels: {
+        mobileNumber: 'Mobile Number',
+        addressGroup: 'Address',
+        line1: 'Address line 1',
+        line2: 'Address line 2 (optional)',
+        locality: 'City',
+        postcode: 'Postcode',
+        deliveryOrderMethod: 'Delivery time',
+        collectionOrderMethod: 'Collection time',
+        dineinOrderMethod: 'Dine in time',
+        tableIdentifier: 'Table number or name'
+    },
+
+    validationMessages: {
+        mobileNumber: {
+            requiredError: 'Please enter your phone number',
+            invalidCharError: 'Your phone number should be at least 10 characters long and shouldn’t contain letters or special characters'
+        },
+        addressLine1: {
+            requiredError: 'Please enter the first line of your address'
+        },
+        locality: {
+            requiredError: 'Please enter your town or city'
+        },
+        postcode: {
+            requiredError: 'Please enter your full UK postcode',
+            invalidCharError: 'This doesn’t look like a UK postcode, can you enter it again please?'
```

Comment:
Shouldn't this to be changed to AU postcode?

Reply:
Good spot! 😬